courses/views.py

```diff
@@ -1,17 +1,25 @@
 """
 Views for powering the Course Catalog API
 """
+import logging
 from six.moves.urllib import parse
 
 from django.core.urlresolvers import reverse
+from django.shortcuts import get_object_or_404
 
+import requests
```

Comment:
Nit: is there any reason why you put the imports
```
from django.core.urlresolvers import reverse
from django.shortcuts import get_object_or_404
```
in a separated group before this?